sdk/keyvault/Azure.Security.KeyVault.Keys/README.md

```diff
@@ -120,24 +120,25 @@ The following section provides several code snippets using the `client` [created
 * [List keys](#list-keys)
 * [Encrypt and Decrypt](#encrypt-and-decrypt)
```

Comment:
Encrypt and Decrypt what?  Should we include "Keys" in this title?

Reply:
We're not encrypting and decrypting keys, but instead using those keys to encrypt and decrypt anything.